modules/tinymce/src/core/test/ts/browser/keyboard/ArrowKeysCefTest.ts

```diff
@@ -146,6 +153,15 @@ UnitTest.asynctest('browser.tinymce.core.keyboard.ArrowKeysCefTest', (success, f
         tinyActions.sContentKeystroke(Keys.down()),
         sAssertScrollCount(2),
         tinyApis.sAssertSelection([ 2, 0 ], 0, [ 2, 0 ], 0)
+      ]),
+      Log.stepsAsStep('TINY-6471', 'Should not throw exception when line below when bogus cef is below', [
+        tinyApis.sSetRawContent('<p><br data-mce-bogus="1"></p><div contenteditable="false" data-mce-bogus="1"  style="user-select: none;"><div contenteditable="false" data-mce-bogus="1"></div></div>'),
+        tinyApis.sSetCursor([ 0, 0 ], 0),
+        sResetKeydownCount,
+        tinyActions.sContentKeystroke(Keys.down()),
+        tinyActions.sContentKeystroke(Keys.down()),
+        // Checking 2 events fired verifies the event handlers finished running, so an exception shouldn't have been raised
```

Comment:
Is this a safe assumption / will this stay a safe assumption? As I understand it, this relies on the fact that we currently execute callbacks in the order that they were added — but is that something that we consider stable to depend on?

Also, should we count keyUp events with this?

Reply:
Yes, this is something that's stable and we'd consider it a breaking change if that did change. I didn't see a need to count keyup, as we aren't doing any custom logic on that atm.